manifest.json

```diff
@@ -0,0 +1,41 @@
+[{
+		"resourceAppId": "00000003-0000-0000-c000-000000000000",
+		"resourceAccess": [{
+				"id": "e1fe6dd8-ba31-4d61-89e7-88639da4683d",
```

Comment:
⛏️ This file is indented with tabs where as the bash script is indented with spaces. It would be best if all the files in the repo used the same indentation type.